reactive/vpe_router.py

```diff
@@ -35,8 +35,10 @@ def validate_config():
     except Exception as e:
         remove_state('vpe.configured')
         set_state('blocked', 'validation failed: %s' % e)
-    else:
+    finally:
+        remove_state('blocked')
```

Comment:
Line 37 set the state. I found setting the config values never cleared the state.